### PR TITLE
(GH-1355) Add information how to request addins

### LINKED ISSUE #1355: Add documentation for Addin requests
We have a repo where people can create issues for addin requests in the cake-contrib org: https://github.com/cake-contrib/addin-requests we should make this a first-class citizen by mentioning on website.

### PR DESCRIPTION
**Pull request by pascalberger**
Add information how to request addins

image

Fixes #1355

### CODE REVIEW

File: input/_ExtensionsPage.cshtml [hunk 1]
```diff
@@ -6,20 +6,16 @@
...
 <div class="alert alert-info">
-    <p>
-        This page contains extensions for Cake.
-        If you want to file an issue for any extension listed here, contact the author of the extension.
-    </p>
-    <p>
-        If you're an extension author and want your extension listed here, follow the instructions <a href="https://github.com/cake-build/website/blob/master/README.md#extensions" target="_blank">here</a>.
-    </p>
+    <i class="fa fa-bug" style="margin-right:20px;" aria-hidden="true"></i>If you want to file an issue for any extension listed here please contact the author of the extension.<br/>
+    <i class="fa fa-hand-pointer-o" style="margin-right:20px;" aria-hidden="true"></i>If you have requests for additional addins <a href="https://github.com/cake-build/cake/discussions?discussions_q=category%3A%22Extension+requests%22" target="_blank">start a discussion</a>.<br/>
```
Comment (augustoproiete) [inline input/_ExtensionsPage.cshtml:17]: I would remove the `target="blank"` of these two links as it's generally a good practice to let the user choose if they want to open a link on the current tab, new tab, new window, etc.

Comment (pascalberger) [reply input/_ExtensionsPage.cshtml:17]: We currently open all external links in a new window and should stick to this for consistency.

**Review by gep13 — APPROVED** [review]
LGTM!

File: input/_ExtensionsPage.cshtml (hunk 1 above)
Comment (pascalberger) [reply input/_ExtensionsPage.cshtml:17]: If we want to change this it should be done for the whole site in a separate issue / PR. And is something we should discuss first.

**Pull request merged**